using-valtimo/document/custom-case-headers.md

```diff
@@ -1,5 +1,9 @@
 # Custom case headers
 
+This page show how a header, that is shown above every case, can be customized.
```

Comment:
```suggestion
This page explains how a header that is shown above every case can be customised.
```

Reply:
@theo-ritense I agree with the edit except customised should be customized. According to our style guide we use American spelling: https://docs.valtimo.nl/readme/style-guide